sdk-commons/src/main/java/org/openmrs/maven/plugins/utility/SDKConstants.java

```diff
@@ -96,7 +96,7 @@ public class SDKConstants {
     public static final String REFAPP_3X_ARTIFACT_ID = "distro-emr-configuration";
     public static final String REFAPP_3X_TYPE = "zip";
 
-    public static final String REFERENCEAPPLICATION_2_4 = "org.openmrs.distro:referenceapplication-package:2.4";
+    public static final String REFERENCEAPPLICATION_3_0 = "org.openmrs:distro-emr-configuration:3.0.0";
```

Comment:
Do we have this version of the O3 reference application?

Reply:
Yes, https://github.com/openmrs/openmrs-distro-referenceapplication/commit/47bf630a26cd33d12fdd583e5e270c1cec33d72d, maybe I can  default it to one of the most recent O3 versions i.e `3.4.0`

Reply:
Is the a reason why you chose an old version instead of the most recent release?

Reply:
No reason

Reply:
I have updated this